### PR TITLE
Xcode 14

### PR DESCRIPTION
**Pull request by watt**
This PR updates Blueprint to build on Xcode 14 and Swift 5.7. It's mostly CI config.

Updating SwiftFormat picked up a few new linting changes.

`xcversion` is [being sunset](https://github.com/xcpretty/xcode-install) and does not support Xcode 14. Replaced it with [xcodes](https://github.com/RobotsAndPencils/xcodes).

There's a new entry in the snapshot test matrix for iOS 16.2. Also deleted the images from iOS 13 & 14 which are now unused.

### CODE REVIEW

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -12,18 +10,21 @@ jobs:
...
-    runs-on: macos-11
+    runs-on: macos-12
 
     strategy:
       fail-fast: false # Don’t fail-fast so that we get all snapshot test changes
       matrix:
         include:
           - sdk: "14.5"
-            destination: platform=iOS Simulator,OS=14.5,name=iPhone 8
+            destination: platform=iOS Simulator,OS=14.5,name=iPad (8th generation)
```
Comment (watt) [inline .github/workflows/tests.yaml:20]: Installing iOS 14.5 with `xcodes` doesn't create an iPhone 8 sim for some reason, so I just picked another 2x device.

File: .github/workflows/tests.yaml [hunk 2]
```diff
@@ -43,7 +47,7 @@ jobs:
     - name: Package snapshot changes
       if: ${{ failure() }}
       run: |
-        git ls-files -mo BlueprintUICommonControls/Tests/Sources/ReferenceImages | xargs tar -cvf snapshot_changes_${{ matrix.sdk }}.tar
+        git ls-files -mo BlueprintUICommonControls/Tests/Sources/Resources/ReferenceImages | xargs tar -cvf snapshot_changes_${{ matrix.sdk }}.tar
```
Comment (watt) [inline .github/workflows/tests.yaml:50]: This fixes the link to download failed snapshot test images.

File: .swiftpm/xcode/xcshareddata/xcschemes/BlueprintUI-Package.xcscheme [hunk 3]
```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Scheme
```
Comment (watt) [inline .swiftpm/xcode/xcshareddata/xcschemes/BlueprintUI-Package.xcscheme:2]: I committed these schemes to work around a bug in Xcode 14. More details linked from the comment in the gitignore file.

File: BlueprintUICommonControls/Tests/Sources/PerformancePlayground.swift [hunk 4]
```diff
@@ -24,7 +24,7 @@ class PerformancePlayground: XCTestCase {
 
     override func invokeTest() {
         // Uncomment this line to run performance metrics, eg in Instruments.app.
-        super.invokeTest()
+        // super.invokeTest()
```
Comment (watt) [inline BlueprintUICommonControls/Tests/Sources/PerformancePlayground.swift:27]: I think this was previously committed accidentally. Disabled again to make the tests run faster.

**Review by kyleve — APPROVED** [review]

**Pull request merged**